README.markdown

```diff
@@ -323,7 +323,9 @@ load-nvmrc() {
     local nvmrc_node_version=$(nvm version "$(cat "${nvmrc_path}")")
 
     if [ "$nvmrc_node_version" != "N/A" ] && [ "$nvmrc_node_version" != "$node_version" ]; then
-      nvm use 
+      nvm use
```

Comment:
hm, yeah that's true. maybe it's best to run the appropriate command. this still needs to be tweaked tho.